<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Testing</class>
 <widget class="QMainWindow" name="Testing">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>682</width>
    <height>535</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(139,233,159);
color: rgb(85, 13, 57);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="answerLabel">
          <property name="font">
           <font>
            <pointsize>20</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Відповіді:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="answersLabel">
          <property name="font">
           <font>
            <pointsize>22</pointsize>
           </font>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLabel" name="emptyLabel">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="timeLabel">
          <property name="font">
           <font>
            <pointsize>20</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Час:</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLCDNumber" name="lcdNumber">
          <property name="segmentStyle">
           <enum>QLCDNumber::Filled</enum>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="tabPosition">
       <enum>QTabWidget::North</enum>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <property name="iconSize">
       <size>
        <width>16</width>
        <height>16</height>
       </size>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Page</string>
       </attribute>
      </widget>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="emptyLabel_2">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="missButton">
        <property name="styleSheet">
         <string notr="true">QPushButton:hover{
	border-width: 0px;
	border-radius: 20%;
}
QPushButton {
	border: 4px solid;
	border-color: rgb(52, 101, 164);
	border-radius: 6px;
	background-color: rgb(189, 214, 254);
	color: rgb(9, 4, 32);
	padding: 10px;
	outline: none;
}

QPushButton:flat {
	border: none; 
}

QPushButton:default {
border-color: rgb(92, 53, 102);
}

QPushButton:disabled{
background-color: rgb(211, 215, 207);
}</string>
        </property>
        <property name="text">
         <string>Пропустити</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="emptyLabel_3">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="answerButton">
        <property name="styleSheet">
         <string notr="true">QPushButton:hover{
	border-width: 0px;
	border-radius: 20%;
}
QPushButton {
	border: 4px solid;
	border-color: rgb(52, 101, 164);
	border-radius: 6px;
	background-color: rgb(189, 214, 254);
	color: rgb(9, 4, 32);
	padding: 10px;
	outline: none;
}

QPushButton:flat {
	border: none; 
}

QPushButton:default {
border-color: rgb(92, 53, 102);
}

QPushButton:disabled{
background-color: rgb(211, 215, 207);
}</string>
        </property>
        <property name="text">
         <string>Відповісти</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="emptyLabel_4">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>682</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
